chaco/grid_mapper.py

```diff
@@ -126,6 +126,10 @@ def map_screen(self, data_pts):
 
         Maps values from data space into screen space.
         """
+        # ensure data_array is an Nx2 ndarray
+        data_pts = array(data_pts)
+        data_pts = data_pts.reshape(-1, 2)
```

Comment:
the `column_stack` call below actually avoids any errors here making this code un-necessary.  Ref:
https://github.com/enthought/chaco/blob/d83e4b8db5bf44be40c450b364ebd159a00c4d62/chaco/tests/test_grid_mapper.py#L36-L42